tests/lib/Configuration/Provider/CKEditorTest.php

```diff
@@ -0,0 +1,454 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\Tests\EzPlatformRichText\Configuration\Provider;
+
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use EzSystems\EzPlatformRichText\Configuration\Provider\CKEditor;
+use EzSystems\EzPlatformRichText\SPI\Configuration\Provider;
+use EzSystems\EzPlatformRichTextBundle\DependencyInjection\Configuration\Parser\FieldType\RichText;
+
+final class CKEditorTest extends BaseProviderTestCase
+{
+    public function createProvider(ConfigResolverInterface $configResolver = null): Provider
+    {
+        return new CKEditor(
+            $configResolver ?? $this->configResolver
+        );
+    }
+
+    public function getExpectedProviderName(): string
+    {
+        return 'CKEditor';
+    }
+
+    /**
+     * @covers \EzSystems\Tests\EzPlatformRichText\Configuration\Provider\CKEditorTest::createProvider
+     */
+    public function testGetConfiguration(): void
+    {
+        $provider = $this->createProvider();
+
+        $this->configResolver
+            ->expects($this->exactly(1))
```

Comment:
It's static, so should be called with `self::` and there's more specific method to achieve the same:
```suggestion
            ->expects(self::once())
```